rewrite-java-core/src/test/java/RetryOptionsConstructorTest.java

```diff
@@ -56,16 +56,20 @@ void testChangeRetryOptionsTypeNoArgInit() {
         @Language("java") String before = "import com.azure.core.http.policy.RetryOptions;import java.time.Duration;import com.azure.core.http.policy.FixedDelayOptions;";
         before += "\npublic class Testing {";
         before += "\n  FixedDelayOptions f = new FixedDelayOptions(3, Duration.ofMillis(50));";
+        before += "\n  String test = \"testing\";";
```

Comment:
Why is this needed?

Reply:
Removed in commit [492daeb](https://github.com/Azure/azsdk-java-rewrite-recipes/pull/14/commits/492daeb48360470829cc54218911b9b3e1758ace)